meta/appveyor.yml

```
@@ -12,6 +12,9 @@ image: Ubuntu
 environment:
   TOOLCHAIN: 1.51.0
 
```

> This only works when we add the secrets to the repo...

```suggestion
  DOCKER_USER:
    secure: xxxxxxx
  DOCKER_PASS:
    secure: yyyyyyy
```
This can be used if someone with the permissions provide the encrypted values. 
https://ci.appveyor.com/tools/encrypt

---

Good, thanks. I can do that.